### PR TITLE
461 msl sources

### LINKED ISSUE #461: Multiple RPS/MSL Sources
- At the moment, we draw RPS/MSL from different sources to the VATSIM METAR server.
- This can lead to inconsistencies when the server is behind RW, or has issues.
- Unify under the VATSIM metar server.

### PR DESCRIPTION
**Pull request by AndyTWF**
Close #461 

- Update MSLs and Regional Pressures to all use the VATSIM METAR server. Rework the code to be nicer to understand.
- Update METARs every 5 minutes and queue jobs for updating MSL/RPS upon completion. These are parsed and stored in the metars table.
- MSL/RPS updates now happen based on the metars returned from the above job.
- Only inform the plugins if something actually changes with regards to RPS/MSL.
- Remove old RPS/MSL update commands from the schedule.
- Remove JSON msl calculation columns in favour of relational approach - easier for querying.

### CODE REVIEW

File: config/metar.php [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 // Config for downloading METARs in the context of MSL and RPS
 return [
-    'vatsim_url' => env('VATSIM_METAR_URL', 'metar.vatsim.net'),
+    'vatsim_url' => env('VATSIM_METAR_URL', 'https://metar.vatsim.net'),
     'regional_url' => env('APP_REGIONAL_PRESSURES_URL', 'https://www.aviationweather.gov/adds/dataserver_current/httpparam?dataSource=metars&requesttype=retrieve&format=xml&hoursBeforeNow=3&mostRecentForEachStation=constraint&stationString=~gb&fields=raw_text,station_id'),
```
Comment (AxonC) [inline config/metar.php:6]: Do you still need this `regional_url` given you remove it from the .env examples?

File: database/migrations/2021_03_23_202453_create_metar_table.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('metars', function (Blueprint $table) {
+            $table->id();
+            $table->text('raw')->comment('The raw METAR text');
+            $table->unsignedInteger('airfield_id')->unique()->comment('The airfield the METAR applies to');
+            $table->unsignedInteger('qnh')
```
Comment (AxonC) [inline database/migrations/2021_03_23_202453_create_metar_table.php:20]: Would there be any merit of storing this in a JSON column so you wouldn't have to add a new column for each kind of new data (I'm possibly thinking of something like LVP detection etc?)

**Review by AxonC — DISMISSED** [review]

File: database/migrations/2021_03_23_202453_create_metar_table.php (hunk 2 above)
Comment (AndyTWF) [reply database/migrations/2021_03_23_202453_create_metar_table.php]: 67e5a27fcad1742e752df8ccf8d112b058755805

File: config/metar.php (hunk 1 above)
Comment (AndyTWF) [reply config/metar.php]: f9ae0f19a16840f54d7f5e3a13ff1696ed1c1fac

File: database/migrations/2021_03_23_202453_create_metar_table.php (hunk 2 above)
Comment (AxonC) [reply database/migrations/2021_03_23_202453_create_metar_table.php]: Love it!

**Review by AxonC — APPROVED** [review]

**Pull request merged**